docs/src/config/ini-config.adoc

```diff
@@ -1296,39 +1132,34 @@ latexmath:[ \frac{X\, inches}{1\, encoder\, count} =
 image::images/encoder-counts-math.png[align="center"]
 --
 
-* 'BIAS = 0.000' - This is used by hm2-servo and some others.
+* `BIAS = 0.000` - This is used by hm2-servo and some others.
   Bias is a constant amount that is added to the output.
   In most cases it should be left at zero.
   However, it can sometimes be useful to compensate for offsets in servo
   amplifiers, or to balance the weight of an object that moves vertically.
   Bias is turned off when the PID loop is disabled, just like all other components of the output.
-* 'P = 50' - The proportional gain for the joint servo.
-  This value multiplies the error between commanded and actual position in machine units, resulting
-  in a contribution to the computed voltage for the motor amplifier.
+* `P = 50` - The proportional gain for the joint servo.
+  This value multiplies the error between commanded and actual position in machine units, resulting in a contribution to the computed voltage for the motor amplifier.
   The units on the P gain are volts per machine unit, e.g., image:images/p-term.png[height=25]
 //latexmath:[$\frac{volt}{mu}$].
-* 'I = 0' - The integral gain for the joint servo.
-  The value multiplies the cumulative error between commanded and actual position in machine
-  units, resulting in a contribution to the computed voltage for the motor amplifier.
+* `I = 0` - The integral gain for the joint servo.
+  The value multiplies the cumulative error between commanded and actual position in machine units, resulting in a contribution to the computed voltage for the motor amplifier.
   The units on the I gain are volts per machine unit second, e.g., image:images/i-term.png[height=25]
 //latexmath:[$\frac{volt}{mu\, s}$].
-* 'D = 0' - The derivative gain for the joint servo.
-  The value multiplies the difference between the current and previous errors, resulting in a
-  contribution to the computed voltage for the motor amplifier.
+* `D = 0` - The derivative gain for the joint servo.
+  The value multiplies the difference between the current and previous errors, resulting in a contribution to the computed voltage for the motor amplifier.
   The units on the D gain are volts per machine unit per second, e.g., image:images/i-term.png[height=25]
 // latexmath:[$\frac{volt}{mu/s}$].
-* 'FF0 = 0' - The 0th order feed forward gain.
+* `FF0 = 0` - The 0th order feed forward gain.
```

Comment:
after this some ``` ` ` ``` changes are missing until STEP_SCALE

Reply:
I found another one "OFFSET_AV_RATIO" :D

Reply:
Done